src/data/styles/header.scss

```diff
@@ -0,0 +1,385 @@
+@import "./variables.scss";
+
+header {
+  background-color: $navy;
+  color: $white;
+  text-align: center;
+  font-size: $font-xs;
+  letter-spacing: 0.2px;
+  line-height: 38px;
+  height: 38px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 82px;
+  width: 100%;
+  padding-left: 64px;
+  padding-right: 64px;
+  color: $white;
+  background: $navbar-gradient;
+  position: absolute;
+  z-index: 10;
+
+  .menu {
+    display: none;
+  }
+
+  ul {
+    list-style-type: none;
+    height: 100%;
+
+    li {
+      display: inline;
+      color: inherit;
+      height: 100%;
+
+      &:hover {
+        border-bottom: 4px solid $grey;
+      }
+    }
+  }
+
+  .navbar-left,
+  .navbar-right {
+    font-size: $font-s;
+    line-height: 77px;
+    align-items: center;
+  }
+
+  .navbar-left {
+    width: 22%;
+    display: flex;
+    justify-content: flex-start;
+
+    i:hover {
+      transform: rotate(-180deg);
+    }
+
+    .nav-stories {
+      margin-left: 51px;
+    }
+  }
+
+  .navbar-right {
+    width: 22%;
+    display: flex;
+    justify-content: space-between;
+    letter-spacing: 0.2px;
+
+    .search {
+      i {
+        font-size: 13px;
+        margin-right: 6.5px;
+      }
+    }
+
+    .user-name {
+      i {
+        font-size: $font-sm;
+        margin-right: 7px;
+      }
+    }
+
+    .quantity {
+      color: $cobalt-blue;
+      opacity: 1;
+      position: relative;
+      font-size: 19px;
+      margin-left: 9px;
+
+      i {
+        color: $white;
+        position: absolute;
+        top: -5px;
+        left: 33%;
+        font-size: $font-xs;
+      }
+    }
+  }
+
+  .brand-info {
+    display: flex;
+    flex-direction: column;
+
+    .logo {
+      height: 26px;
+      background: url("../assets/header/logo.png") no-repeat center;
+    }
+
+    .brand-name {
+      color: $white;
+      padding-top: 10px;
+    }
+  }
+
+  &:hover {
+    color: $black;
+    background-color: $white;
+    background-image: none;
+
+    .logo {
```

Comment:
Whenever possible, it's better to use SVGs for hover effects where you want to change the color of a logo